test/framework/toolchain.py

```diff
@@ -1161,6 +1161,44 @@ def test_standalone_iccifort(self):
         # no dependencies found in iccifort module
         self.assertEqual(tc.toolchain_dep_mods, [])
 
+    def test_standalone_iccifortcuda(self):
+        """Test whether standalone installation of iccifortcuda matches the iccifortcuda toolchain definition."""
+
+        tc = self.get_toolchain('iccifortcuda', version='2019a')
+        tc.prepare()
+        self.assertEqual(tc.toolchain_dep_mods, ['icc/2018.1.163', 'ifort/2018.1.163', 'CUDA/9.1.85'])
+        self.modtool.purge()
+
+        for key in ['EBROOTICC', 'EBROOTIFORT', 'EBVERSIONICC', 'EBVERSIONIFORT', 'EBROOTCUDA', 'EBVERSIONCUDA']:
+            self.assertTrue(os.getenv(key) is None)
+
+        # install fake iccifortcuda module with no dependencies
+        fake_iccifortcuda = os.path.join(self.test_prefix, 'iccifortcuda', '2019a')
+        write_file(fake_iccifortcuda, "#%Module")
+        self.modtool.use(self.test_prefix)
+
+        # toolchain verification fails because icc/ifort are not dependencies of iccifort modules,
+        # and corresponding environment variables are not set
+        error_pattern = "List of toolchain dependency modules and toolchain definition do not match"
+        self.assertErrorRegex(EasyBuildError, error_pattern, tc.prepare)
+        self.modtool.purge()
+
+        # make iccifortcuda module set $EBROOT* and $EBVERSION* to pass toolchain verification
+        fake_iccifortcuda_txt = '\n'.join([
+            "#%Module",
+            'setenv EBROOTICC "%s"' % self.test_prefix,
+            'setenv EBROOTIFORT "%s"' % self.test_prefix,
```

Comment:
@akesandgren This doesn't correctly reflect how an actual `iccifortcuda` would look like, that'll always be a bundle with a `module load iccifort` and `module load CUDA` (as opposed to `iccifort` which could be a stand-alone installation not loading any modules).

If you change this test to reflect how things correctly, then there should be no need to tweak `is_dep_in_toolchain_module` in `iccifortcuda.py` at all, you only had to do that to make an `iccifortcuda` module that doesn't have any `module load` statements in it work, but that shouldn't happen.